tests/specs/npm/bytes_and_text_imports/__test__.jsonc

```diff
@@ -0,0 +1,19 @@
+{
+  "tempDir": true,
+  "tests": {
+    "run": {
+      "args": "run main.ts",
+      "output": "run.out"
+    },
+    "compile": {
+      "steps": [{
+        "args": "compile --unstable-raw-imports --output bin main.ts",
```

Comment:
Not super important - but this flag is not needed because there's `deno.json`, right? To test that the flag has effect you'd have to use `--no-config`, I think?

Reply:
Oh, yeah I just accidentally did this. I'm too lazy to do another ci run to fix it now 😄